binrz/rizin/rizin.c

```diff
@@ -41,6 +41,7 @@ static int rz_main_rzpipe(int argc, const char **argv) {
 		eprintf("Error: RZ_PIPE_(IN|OUT) environment not set\n");
 		eprintf("Usage: rizin -c '!*rzp x' # run commands via rzpipe\n");
 		rc = 1;
+		rc = 1;
```

Comment:
isn't that dupe?

Reply:
Yes, i want to generate a new warning to check if it is correctly process as an error